src/components/Header.tsx

```diff
@@ -48,7 +48,7 @@ export const Header = () => {
 
         <ul className="flex space-x-4">
           <li>
-            <button className="px-4 py-2 text-black rounded hover:bg-A6EDF2 hover:text-white font-sans text-base dark:text-white">
+            <button className="drop-shadow-[0_1.2px_1.2px_rgba(0,0,0,0.8)] px-4 py-2 text-black rounded hover:bg-A6EDF2 hover:text-white font-sans text-base dark:text-white">
```

Comment:
Hey @Jihnboii great start! If you look closely, you'll notice that the drop shadow you applied is to the button text, but the Figma header has it on the div. Instead, try applying it to the top div in the Header component. Also, check out how they've achieved the shadow effect in [Figma](https://www.figma.com/file/jl3d5wO6lGLwKhXvrD4kBl/Design-Implementation?type=design&node-id=202-2&mode=design&t=MBrdsy6kcPebByd9-0). Instead of drop shadow, they're using a black stroke with 60% opacity. You could get a similar effect by styling the top div's border in [Tailwind](https://tailwindcss.com/docs/border-radius).

Do you have access to LL's Figma prototypes?

Reply:
Yes I just received access to LL's Figma prototypes